main/src/cgeo/geocaching/address/AddressListActivity.java

```diff
@@ -66,4 +67,9 @@ public void onClickAddress(final Address address) {
         finish();
     }
 
+    @Override
+    public void onClickMapIcon(final Address address) {
+        DefaultMap.startActivityCoords(this, new Geopoint(address.getLatitude(), address.getLongitude()), null, null);
```

Comment:
This method is tailored to display an arbitrary point on a map, not really what your intention is.
As there is currently no method available to achieve what you want, you need to create a new one (which likely includes the creation of a new MapOptions contructor).

Reply:
Created new intent, maptoptions etc. to start with a live map around the coords. (see commit 42a0377)

Seems to work fine generally, BUT after loading caches (live & stored) they get shown and instantly vanish again ?!

Reply:
> Seems to work fine generally, BUT after loading caches (live & stored) they get shown and instantly vanish again ?!

That sounds like one of the regressions we have on the new map.

Reply:
@rsudev 
There seems to be some racing condition, because if I step through it step by step using breakpoints in
- `DefaultMap#43:startActivityGeoCode(...)` (to dive into) and
- `StoredCachesOverlay#66:if(moved) ...` (to see status) 

it works fine (meaning: loaded caches get displayed and do not vanish instantly [*]).
Sadly I do not know enough about Java and concurrent programming to dive further into this...

[*] There is a follow up regression: If I click on a cache, a popup gets shown (ok). If I close this popup by using the back button, the cache vanishes from map view. This is independent from whether or not I save this cache in between.

Reply:
> @moving-bits Which kind of map did you use when you saw the mentioned problems (Google, OSM)?

@Lineflyer I can only test with OSM since my Google Maps keys are not working with c:geo.

Reply:
Are you sure those problems were not already present without your changes here?

Reply:
I cannot say for sure - I had the problem with loosing the display of loaded caches already before, frequently, but not always. On the other hand, with my change I could not get the emulator to make the loaded caches stay displayed at all :-( 
(Have not tested the APK on a physical device yet, though, don't have a spare one.)

Reply:
Ok, I found a physical device to test this on. Works like a charm with the new map.